hack/update-openapi.sh

```diff
@@ -18,6 +25,7 @@ openapi-gen \
   $@
 [ -e ./v2 ] && rm -rf v2
 
-export GO111MODULE=off
+export GO111MODULE=on
```

Comment:
What does this change do?

Reply:
even with GOPATH set when GO111MODULE=off , build doesn't work and failed ,but works when GO111MODULE=on